From 1e9073ae96ede82e475d14bc90023b1a0055ee94 Mon Sep 17 00:00:00 2001
From: Daniel Vetter <daniel.vetter@ffwll.ch>
Date: Wed, 10 Jul 2013 14:11:41 +0200
Subject: drm: kill dev->buf_readers and dev->buf_writers

Again totally unused, so just remove them.

Signed-off-by: Daniel Vetter <daniel.vetter@ffwll.ch>
Signed-off-by: Dave Airlie <airlied@gmail.com>
(cherry picked from commit 494f38e4e0c5ffca110e361cd3391f25313b52c7)
Signed-off-by: Darren Hart <dvhart@linux.intel.com>
---
 drivers/gpu/drm/drm_fops.c | 2 --
 include/drm/drmP.h         | 2 --
 2 files changed, 4 deletions(-)

diff --git a/drivers/gpu/drm/drm_fops.c b/drivers/gpu/drm/drm_fops.c
index cdda3951ff03..2f86ccb86dfc 100644
--- a/drivers/gpu/drm/drm_fops.c
+++ b/drivers/gpu/drm/drm_fops.c
@@ -81,8 +81,6 @@ static int drm_setup(struct drm_device * dev)
 	dev->if_version = 0;
 
 	dev->buf_async = NULL;
-	init_waitqueue_head(&dev->buf_readers);
-	init_waitqueue_head(&dev->buf_writers);
 
 	DRM_DEBUG("\n");
 
diff --git a/include/drm/drmP.h b/include/drm/drmP.h
index 952478cd3ec3..a47fec92c6ca 100644
--- a/include/drm/drmP.h
+++ b/include/drm/drmP.h
@@ -1171,8 +1171,6 @@ struct drm_device {
 	/*@} */
 
 	struct fasync_struct *buf_async;/**< Processes waiting for SIGIO */
-	wait_queue_head_t buf_readers;	/**< Processes waiting to read */
-	wait_queue_head_t buf_writers;	/**< Processes waiting to ctx switch */
 
 	struct drm_agp_head *agp;	/**< AGP data */
 
-- 
1.8.5.rc3

